### ISSUE TITLE
Enhance TEAM Engine to evaluate if core conformance classes are configured

### ISSUE THREAD

**Issue by dstenger** [opengeospatial/teamengine#284]
**Current behaviour:**

The EARL/HTML report states the following:
```
Passed core (Can be certified): Yes/No
```
This is also the case if no BasicConformanceClasses are configured. In this case, the information is wrong as it cannot be evaluated whether core is passed.

___

**Expected behaviour:**

If no BasicConformanceClasses are configured, the EARL/HTML report states the following:
```
Passed core (Can be certified): This test suite does not support evaluation of core conformance classes. Please contact the CITE team if you have any questions regarding the core conformance classes.
```

___

Benefit of this enhancement: The new HTML report never includes erroneous informations (right now, ```Passed core (Can be certified)``` is filled with Yes or No) and is always correct. This allows to move the newest version of TEAM Engine to production even if not all test suites have been updated.

**Comment by dstenger** [comment]
@bermud What do you think of this idea? This speeds up the process of moving new TEAM Engine (v5+) to production very much as it is not possible any more that erroneous reports are generated.

@keshav-nangare Can you please estimate how complex this enhancement is and what time you need to implementat this?

**Comment by keshav-nangare** [comment]
@dstenger 

I have updated the teamengine according to the suggestion, now the teamengine will evaluate if conformance classes are configure or not and based on that it will show correct result.

If the no basic conformance classes are configured then it will display the message as suggested in above [comment](https://github.com/opengeospatial/teamengine/issues/284#issue-282050432).



Fixed with this commit: 516356b433aa9d4fee23099a9e4a0312c7804d5c [PR: #290]

**Comment by lgoltz** [comment]
I've just tested the changes with the local docker environment 
 *  ETS WMTS 1.0 NSG v0.1 (SUT:  http://ips.terrapixel.com/terrapixel/cubeserv.cgi?SERVICE=WMTS&REQUEST=GetCapabilities) and
 * ETS WFS 2.0 v1.28-SNAPSHOT (SUT: http://cite.deegree.org/deegree-webservices-3.4-RC3/services/wfs200?service=WFS&request=GetCapabilities)

For both the conformance classes are configured. I expect a result like `Passed core (Can be certified): Yes` (or no), but it is: 

![result](https://user-images.githubusercontent.com/3007848/34821475-1b4bf8ec-f6c4-11e7-8dcf-aaf53285990b.png)

Furthermore the formatting of the text makes it difficult to understand.

**Comment by lgoltz** [comment]
The pull request has conflicts which must be resolved first.

**Comment by keshav-nangare** [comment]
@lgoltz 

The conflicts of this branch are resolved. Can you please merge this PR?

**Comment by lgoltz** [comment]
@keshav-nangare Thanks for your investigation. Can you also take a look into https://github.com/opengeospatial/teamengine/issues/284#issuecomment-356900372?

**Comment by keshav-nangare** [comment]
I have merged this branch[feature-issue#284] locally to master and tested with wfs20[1.28- SNAPSHOT] but it's not reproducible. Updated the text with correct indentation.

**Comment by dstenger** [comment]
**Test with Docker environment [1]**

* ETS WMTS 1.0 NSG (SUT: http://ips.terrapixel.com/terrapixel/cubeserv.cgi?SERVICE=WMTS&REQUEST=GetCapabilities) and
* ETS WFS 2.0 (SUT: http://cite.deegree.org/deegree-webservices-3.4-RC3/services/wfs200?service=WFS&request=GetCapabilities)

Both tests returned the expected results (```Passed core (Can be certified): No``` and ```Passed core (Can be certified): Yes```).

* ETS GML 3.2 (SUT: http://cite.deegree.org/deegree-webservices-3.4-RC3/services/gml321?service=WFS&request=GetFeature&Version=2.0.0&typenames=app:Autos)

Returns expected result: ```Passed core (Can be certified): This test suite does not support evaluation of basic conformance classes. Please contact the CITE team if you have any questions regarding the basic conformance classes.```

[1] https://github.com/opengeospatial/teamengine-docker
